### ISSUE TITLE
Some appearance methods don't work.

### ISSUE THREAD

**Issue by capecrawler** [zendesk/zendesk_sdk_chat_ios#44]
I am trying to override the appearance of the chat ui and I think some are not working or I am doing something wrong. I even tried editing the sample project.
Title and message text color should be blue.

```
...
    [[ZDCLoadingErrorView appearance] setTitleFont:[UIFont boldSystemFontOfSize:25.0f]];
    [[ZDCLoadingErrorView appearance] setTitleColor:[UIColor blueColor]];
    [[ZDCLoadingErrorView appearance] setMessageFont:[UIFont systemFontOfSize:11.0f]];
    [[ZDCLoadingErrorView appearance] setMessageColor:[UIColor blueColor]];
    [[ZDCLoadingErrorView appearance] setButtonFont:[UIFont boldSystemFontOfSize:15.0f]];
    [[ZDCLoadingErrorView appearance] setButtonTitleColor:[UIColor whiteColor]];
    [[ZDCLoadingErrorView appearance] setButtonBackgroundColor:[UIColor greenColor]];
...
```
![zopim_custom](https://cloud.githubusercontent.com/assets/1147223/11833234/a9e02056-a3fb-11e5-9aa6-c87a77f06ec1.png)

**Comment by tecknut** [comment]
@capecrawler Thanks for pointing this out. This is in fact a bug in the `ZDCLoadingErrorView`. We have an issue to track this and it will be out in our next release. 

We have just released a new version today which will not have the fix in it, just to be clear.

Thanks again for spotting this.

**Comment by marcoscurvello** [comment]
What's the status on this issue?

**Comment by RonanMcH** [comment]
Hi @marcoscurvello ,
We have fixed these issues internally and we shall be releasing soon with them fixed. We notify you and close this issue once we have done so.
Thanks for bringing these bugs to our attention,
Ronan.

**Comment by oarrabi** [comment]
Hello @capecrawler, @marcoscurvello 

We just released `ZDCChat` version 1.2.0.1 which has a fix to this issue.
Please check it out and let us know.

Thanks!

**Comment by oarrabi** [comment]
I will close this issue now, if it stills persist, please don't hesitate to reopen it.
